Sources/KeyboardShortcuts/Localization/ar.lproj/Localizable.strings

```diff
@@ -0,0 +1,5 @@
+"record_shortcut" = "سجل اختصار";
```

Comment:
```suggestion
"record_shortcut" = "سجل اختصاراً";
```

Not familiar with the context here but this string is used when we want to record a new keyboard shortcut, right?